### PR TITLE
Rr distribution

### PR DESCRIPTION
**Pull request by rdr335**
I edited my mean and standard deviation by calculating them using python. the standard deviation is reported to the 4th decimal place because these places remained constant after increasing the sample size to very large numbers.

### CODE REVIEW

**Comment by cranmer** [comment]
You are supposed to calculate the std. dev. and mean using calculous, not by sampling the pdf.

**Comment by rdr335** [comment]
I'm having trouble with the integration for this particular formula. :/

**Comment by rdr335** [comment]
is that right? (also having trouble using your testing code. I don't know where/ how to run it. when I put it in my python ide it had an error regarding the basedistribution thing. but I don't know enough about python to fix it).

File: distributions/Dist_rdr335.py [hunk 1]
```diff
@@ -1,20 +1,21 @@
 import numpy as np
 from base_distribution import BaseDistribution
 
+c = np.sqrt(2/np.pi)
```
Comment (cranmer) [inline distributions/Dist_rdr335.py]: defining a constant outside of the code will cause problems.
You can add `self.c = np.sqrt(2/np.pi)` in the `__init__` method similar to `self._x_max`

**Pull request closed without merging**